### PR TITLE
base64: add German translation

### PR DESCRIPTION
**Pull request by xRealNeon**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/base64.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# base64
+
+> Kodierung oder Dekodierung von Dateien oder Standardeingaben in und aus Base64, zur Standardausgabe (stdout).
```
Comment (kbdharun) [inline pages.de/common/base64.md:3]:
```suggestion
> Kodierung oder Dekodierung von Dateien oder Standardeingaben in/aus Base64, zur Standardausgabe.
```
Would this be better? What do you think?

> Note: I am not a German speaker. I used Google translate to validate this translation. My suggestion is based on the English page.

Comment (xRealNeon) [reply pages.de/common/base64.md:3]: Yes this seems better

File: pages.de/common/base64.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Kodiert den Inhalt einer Datei als base64 und schreibt das Ergebnis nach stdout:
+
+`base64 {{datei_name}}`
+
+- Dekodiert den Inhalt einer Datei als base64 und schreibt das Ergebnis nach stdout:
+
+`base64 --decode {{datei_name}}`
+
+- Kodiert von stdin:
```
[the remark this answers is not in the document]
Comment (xRealNeon) [reply pages.de/common/base64.md:14]: It can be both but "Kodieren" is maybe nicer 👍

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

File: pages.de/common/base64.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`{{ein_kommando}} | base64`
+
+- Dekodiert von stdin:
```
[the remark this answers is not in the document]
Comment (pixelcmtd) [reply pages.de/common/base64.md]: Yes, but no. The style guide says that we should use the imperative so it has to be `Dekodiere`.

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages.de/common/base64.md]: Thanks for the information. I will check for the imperative mode in future.

**Pull request merged**